osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapLeaderboard.cs

```diff
@@ -281,5 +290,13 @@ public void SetRetrievalState(PlaceholderState state)
                 PlaceholderState = state;
             }
         }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Dependencies.Cache(dialogOverlay);
```

Comment:
use `[Cached]` attribute:
```csharp
    [Cached]
    private readonly DialogOverlay dialogs;
```

Reply:
Thanks for the suggestion. I have made the changes accordingly.